packages/two_dimensional_scrollables/test/table_view/table_span_test.dart

```diff
@@ -177,6 +177,669 @@ void main() {
+            // Rows first, bottom to top for reversed axis
+            ..rect(
+              rect: const Rect.fromLTRB(0.0, 500.0, 1000.0, 600.0),
+              color: const Color(0xffbbdefb),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(0.0, 400.0, 1000.0, 500.0),
+              color: const Color(0xffbbdefb),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(0.0, 300.0, 1000.0, 400.0),
+              color: const Color(0xffbbdefb),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(0.0, 200.0, 1000.0, 300.0),
+              color: const Color(0xffbbdefb),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(0.0, 100.0, 1000.0, 200.0),
+              color: const Color(0xffbbdefb),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(0.0, 0.0, 1000.0, 100.0),
+              color: const Color(0xffbbdefb),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(0.0, -100.0, 1000.0, 0.0),
+              color: const Color(0xffbbdefb),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(0.0, -200.0, 1000.0, -100.0),
+              color: const Color(0xffbbdefb),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(0.0, -300.0, 1000.0, -200.0),
+              color: const Color(0xffbbdefb),
+            )
+            // Columns next
+            ..rect(
+              rect: const Rect.fromLTRB(0.0, -300.0, 100.0, 600.0),
+              color: const Color(0xffe1bee7),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(100.0, -300.0, 200.0, 600.0),
+              color: const Color(0xffe1bee7),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(200.0, -300.0, 300.0, 600.0),
+              color: const Color(0xffe1bee7),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(300.0, -300.0, 400.0, 600.0),
+              color: const Color(0xffe1bee7),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(400.0, -300.0, 500.0, 600.0),
+              color: const Color(0xffe1bee7),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(500.0, -300.0, 600.0, 600.0),
+              color: const Color(0xffe1bee7),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(600.0, -300.0, 700.0, 600.0),
+              color: const Color(0xffe1bee7),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(700.0, -300.0, 800.0, 600.0),
+              color: const Color(0xffe1bee7),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(800.0, -300.0, 900.0, 600.0),
+              color: const Color(0xffe1bee7),
+            )
+            ..rect(
+              rect: const Rect.fromLTRB(900.0, -300.0, 1000.0, 600.0),
+              color: const Color(0xffe1bee7),
+            ));
```

Comment:
I wonder if this formats slightly nicer (i.e. with less indentation) if you place a comma between these two parentheses:

```suggestion
            ),);
```

Here and for the ones below as well.